README.md

```diff
@@ -1 +1,16 @@
-TBD
+<img src="./images/community-logo.png" alt="Eiffel Community Repository Template" width="350"/>
+
+# Eiffel Community
+This repository contains common Eiffel resources and information that and not does belong to a specific Eiffel product. General issues that concerns all repositories can be posted and discussed in this repository as well.
```

Comment:
The first sentence reads a bit funny, maybe it should be "... that does not ..."